study-builder/tenants/testboston/pages/login.html

```diff
@@ -84,37 +84,66 @@
     var headerText = "";
     var studyColor = '#2760b2';   // button primary color
     var studyHelpEmail = '##STUDY_EMAIL##';
-
-    var title = 'Welcome!';
-    if (mode === 'login') {
-        title = 'Please Sign In';
-    }
-    else if (mode === 'signup') {
-        title = 'Create your account';
-    }
-
     var loginUrl = config.callbackURL + '/login';
     var registerUrl = '##BASE_URL##';
```

Comment:
```suggestion
    var registerUrl = '##BASE_URL##' + '/join';
```

Reply:
🤔 Hmmm, this URL has been pointing folks back to the homepage and no one has complained about it so far. Adding `/join` will change the user experience. Maybe we should hold off on that change until either Jen/Erin/Kiara asks for it?

Reply:
Okay